migrate/v0_15/migrate.go

```diff
@@ -191,3 +192,8 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 	return v0_15committee.NewGenesisState(
 		genesisState.NextProposalID, committees, proposals, votes)
 }
+
+// Swap migrates new swap module state to a v0.15 swap genesis state
+func Swap(genesisState v0_15swap.GenesisState) v0_15swap.GenesisState {
```

Comment:
Looks like the Swap state isn't called from MigrateAppState yet

Reply:
fixed in https://github.com/Kava-Labs/kava/pull/925/commits/d202e32a2eb3f245ad1d3c210f12025c8fc6585a